src/java/main/org/apache/jute/Utils.java

```diff
@@ -190,19 +190,32 @@ static String fromCSVString(String s) throws IOException {
     }
     
     /**
+     * convert byte array to a string in hex format
      * 
-     * @param s 
-     * @return 
+     * @param barr
+     * @return
      */
-    static String toXMLBuffer(byte barr[]) {
```

Comment:
@afine I changed the code to use DatatypeConverter.printHexBinary(barr). but the tests keep failing. 
I take a look at the failures. it's very wired, because should be caused by this change.

Also I got different test cases failure in local and in hadoop jenkins report. 

Not sure how to trigger hadoop jenkins job to run tests, so no way to paste it here.